addons/stock/models/procurement.py

```diff
@@ -11,20 +11,34 @@
 from odoo.osv import expression
 from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT, float_compare, float_round
 
-import logging
+from odoo.exceptions import UserError
 
+import logging
 _logger = logging.getLogger(__name__)
 
-class ProcurementGroup(models.Model):
-    _inherit = 'procurement.group'
-
-    partner_id = fields.Many2one('res.partner', 'Partner')
-
 
 class ProcurementRule(models.Model):
-    """ Pull rules """
-    _inherit = 'procurement.rule'
-
+    ''' A rule describe what a procurement should do; produce, buy, move, ... '''
```

Comment:
According PEP257, docstring should be with `"""` (double instead of sinle quotes)